tests/unit/helm/cas-deployment_test.go

```diff
@@ -497,7 +497,7 @@ func (s *deploymentCasTemplateTest) TestContainerEnv() {
 		{
 			"multipleLicenses",
 			map[string]string{
-				"fiftyoneLicenseSecrets[0]": "fiftyonelicense",
+				"fiftyoneLicenseSecrets[0]": "fiftyone-license",
```

Comment:
this was intentionally set to `fiftyonelicense` to test scenarios where `fiftyone-license` wasn't used - the default tests use the default case.

Reply:
Thanks for pointing this out. I undid this change.